### ISSUE TITLE
ScreenRecordingSample doesn´t seem to able to record someone playing a game

### ISSUE THREAD

**Issue by RazorPT** [saki4510t/ScreenRecordingSample#3]
Just noticed that the library works flawlessly now while recording the screen while on android but if i go play a game that uses opengl (I tried Angy Birds and the hidden Lollipop Game) when i stop recording the app freezes and there isn´t any final video. Must be something about the EGL/opengl you used on glUtils? Any ideas?

Can you please test that on your nexus 9?

**Comment by RazorPT** [comment]
Okay just tested the version before you add the workaround for the flickering on Nexus 5 after pause and that version can record someone playing a game so its an issue with the glUtils.

**Comment by saki4510t** [comment]
I'm sorry answer is delayed I'm busy and I don't have enough time to confirm for these days.
I tested on my Nexus9 with Angry bird and can recorded well. But on Nexus5, GALAXY S5 it sometimes failed. As my investigation, it comes from race condition between SurfaceTexture callback and drawing runnable when foreground app has lot of works on cpu/gpu. I added some modification to solve that and I assume it also work on your device.
